tools/deployment/package-helm/test-multi-dedicated.sh

```
@@ -0,0 +1,105 @@
+echo "Installing Helm chart..."
+helm uninstall test --ignore-not-found 2>/dev/null || true
+sleep 2
+helm install test "${script_dir}" \
+    --set "distributed=true" \
+    --set "compressionWorker.replicas=${COMPRESSION_WORKER_REPLICAS}" \
+    --set "compressionWorker.scheduling.nodeSelector.yscope\.io/nodeType=compression" \
+    --set "queryWorker.replicas=${QUERY_WORKER_REPLICAS}" \
+    --set "queryWorker.scheduling.nodeSelector.yscope\.io/nodeType=query" \
+    --set "reducer.replicas=${REDUCER_REPLICAS}"
+
+wait $SAMPLE_DOWNLOAD_PID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)